core/src/main/java/edu/wpi/grip/core/sources/IPCameraFrameGrabber.java

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
Also, IPCameraFrameGrabber isn't a source, It should probably go in a different package.